ledger-core/v2/application/testutils/launchnet/launchnet.go

```diff
@@ -127,7 +127,7 @@ func launchnetPath(a ...string) (string, error) {
 	cwdList := strings.Split(cwd, "/")
 	var count int
 	for i := len(cwdList); i >= 0; i-- {
-		if cwdList[i-1] == "insolar" && cwdList[i-2] == "insolar" {
+		if cwdList[i-1] == "v2" && cwdList[i-2] == "ledger-core" {
```

Comment:
this line can access cwdList[-1] and cwdList[-2]. I believe for condition should be chaned.

Reply:
yep